fix: Button Group Rehaul (#1430)
appsmithorg/appsmith-docs: website/docs/reference/widgets/button-group/README.md

---
description: Learn how to use the Button Group widget for selecting multiple items from predefined choices.
---
# Button Group

This page provides information on using the Button Group widget, which allows you to create a group of buttons, including menu buttons with drop-down items.

<VideoEmbed host="youtube" videoId="oWGK1s5nGns" title="How to use Button Group Widget" caption="How to use Button Group Widget"/>

## Content properties

These properties are customizable options present in the property pane of the widget, allowing users to modify the widget according to their preferences.

### Data

#### Buttons `string`

<dd>

> Displays a list of available items. You can rearrange the items and configure them by clicking on the ⚙️ gear icon. See the [Button](/reference/widgets/button-group/buttons) reference guide for configuring items.

Reviewer: ```suggestion
Specify the buttons available in the group. You can rearrange the buttons and configure them by clicking the gear icon ⚙️. See the [Buttons](/reference/widgets/button-group/buttons) reference guide for configuring individual buttons in the group.
```
